frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.tsx

```diff
@@ -421,6 +448,15 @@ export const FeatureLifecycleTooltip: FC<{
                                 project={project}
                             />
                         ) : null}
+                        {(stage.name === 'initial' ||
```

Comment:
can you add a screenshot to the PR so that we can see the UI part being affected?